DuckDuckGo/Autoconsent/AutoconsentUserScript.swift

```diff
@@ -229,7 +222,7 @@ extension AutoconsentUserScript {
             return
         }
 
-        if preferences.autoconsentPromptSeen == true && preferences.autoconsentEnabled == false {
```

Comment:
`autoconsentPromptSeen` was set to `true` after user enabled the feature (https://github.com/duckduckgo/iOS/pull/2373/files#diff-d04b75c0b651987027e4a487112e5facbd38ff7c45b585c5e202b118eaf68b3dL71).